acquisitions/src/main/resources/thunderjet/mod-consortia/features/tenant.feature

```diff
@@ -106,42 +127,108 @@ Feature: Tenant object in mod-consortia api tests
     When method PUT
     Then status 400
     And match response == { errors: [{ message: 'Request body tenantId and path param tenantId should be identical', type: '-1', code: 'VALIDATION_ERROR' }] }
 
     # cases for 404
-    # attempt to update a tenant for non-existing consortium
-    Given path 'consortia', 'd9acad2f-2aac-4b48-9097-e6ab85906b25', 'tenants', '1234'
-    And request { id: '12345', code: 'ABD', name: 'test', isCentral: false }
+    # attempt to update the tenant for non-existing consortium
+    Given path 'consortia', 'd9acad2f-2aac-4b48-9097-e6ab85906b25', 'tenants', centralTenant
+    And request { id: '#(centralTenant)', code: 'ABD', name: 'test', isCentral: true }
     When method PUT
     Then status 404
     And match response == { errors: [{message: 'Object with consortiumId [d9acad2f-2aac-4b48-9097-e6ab85906b25] was not found', type: '-1', code: 'NOT_FOUND_ERROR'}] }
 
     # attempt to update non-existing tenant
     Given path 'consortia', consortiumId, 'tenants', '12345'
-    And request { id: '12345', code: 'ABD', name: 'test', isCentral: false }
+    And request { id: '12345', code: 'ABD', name: 'test', isCentral: true }
     When method PUT
     Then status 404
     And match response == { errors: [{message: 'Object with id [12345] was not found', type: '-1', code: 'NOT_FOUND_ERROR'}] }
 
+    # cases for 422
+    # attempt to update the tenant without an id
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { code: 'ABC', name: 'test', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'id' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without a code
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', name: 'test', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'code' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without a name
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', code: 'ABC', isCentral: true }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'name' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+    # attempt to update the tenant without isCentral
+    Given path 'consortia', consortiumId, 'tenants', centralTenant
+    And request { id: '#(centralTenant)', code: 'ABC', name: 'test' }
+    When method PUT
+    Then status 422
+    And match response == { errors: [{ message: "'isCentral' validation failed. must not be null", type: '-1', code: 'tenantValidationError'}] }
+
+#    # attempt to update the tenant with a name that has length more than 150 characters and a code with more than 3 characters
```

Comment:
Note for reviewers: new tech debt story created to fix mentioned issue: https://issues.folio.org/browse/MODCON-64 
After fixing this issue we will uncomment these tests